lib/Dialect/Torch/Transforms/DecomposeComplexOps.cpp

```diff
@@ -2101,6 +2101,29 @@ class DecomposeAtenNumpyTOp : public OpRewritePattern<AtenNumpyTOp> {
 };
 } // namespace
 
+namespace {
+class DecomposeAtenNormScalarOptDimOp
+    : public OpRewritePattern<AtenNormScalarOptDimOp> {
+public:
+  using OpRewritePattern::OpRewritePattern;
+  LogicalResult matchAndRewrite(AtenNormScalarOptDimOp op,
+                                PatternRewriter &rewriter) const override {
+    Location loc = op.getLoc();
+    Value self = op.self();
+    Value p = op.p();
+    Value result  = op.result();
+    BaseTensorType tensorType = self.getType().cast<BaseTensorType>();
+    Value abs = rewriter.create<AtenAbsOp>(loc, tensorType, self);
+    Value exp = rewriter.create<AtenExpOp>(loc, tensorType, p);
```

Comment:
`ExpOp` is the exponential function $e^x$, I think you want `PowOp`

Reply:
It seems like there is no support for AtenPowOp.